apis/core/lib/domain/column-mapping/create.ts

```diff
@@ -73,3 +62,71 @@ export async function createColumnMapping({
 
   return columnMapping.pointer
 }
+
+interface CreateLiteralColumnMappingCommand {
+  targetProperty: Term
+  table: Table
+  resource: GraphPointer
+  store: ResourceStore
+}
+
+async function createLiteralColumnMapping({ targetProperty, table, resource, store }: CreateLiteralColumnMappingCommand): Promise<LiteralColumnMapping> {
```

Comment:
Here also I would consider moving these two functions in separate module or modules